app/components/Button.tsx

```diff
@@ -0,0 +1,37 @@
+import { cva, type VariantProps } from "class-variance-authority";
```

Comment:
Could you add a small description and motivation to the PR description as to why you picked this library and what we will use it for?

Reply:
Sure. Added.